Parser/Client/AbstractClientParser.php

```diff
@@ -82,6 +82,43 @@ public static function getAvailableClients(): array
                 continue;
             }
 
+            if ('Microsoft Office $1' === $regex['name']) {
+                $clients = [
+                    'Microsoft Office Access', 'Microsoft Office Excel', 'Microsoft Office OneDrive for Business',
+                    'Microsoft Office OneNote', 'Microsoft Office PowerPoint', 'Microsoft Office Project',
+                    'Microsoft Office Publisher', 'Microsoft Office Visio', 'Microsoft Office Word',
+                ];
+
+                foreach ($clients as $client) {
+                    $names[] = $client;
+                }
+
+                continue;
+            }
+
+            if ('Podkicker$1' === $regex['name']) {
+                $clients = ['Podkicker', 'Podkicker Pro', 'Podkicker Classic'];
+
+                foreach ($clients as $client) {
+                    $names[] = $client;
+                }
+
+                continue;
+            }
+
+            if ('radio.$1' === $regex['name']) {
+                $clients = [
+                    'radio.at', 'radio.de', 'radio.dk', 'radio.es', 'radio.fr', 'radio.it', 'radio.pl', 'radio.pt',
+                    'radio.se', 'radio.net',
+                ];
+
+                foreach ($clients as $client) {
+                    $names[] = $client;
+                }
+
+                continue;
+            }
+
             $names[] = $regex['name'];
         }
```

Comment:
I suggest just combining the array instead of 3 cycles

```php
    public static function getAvailableClients(): array
    {
        $instance = new static(); // @phpstan-ignore-line
        $regexes  = $instance->getRegexes();
        $names    = [];

        foreach ($regexes as $regex) {
            if (\strpos($regex['name'], '$1') !== false) {
                continue;
            }

            $names[] = $regex['name'];
        }

        if (static::class === MobileApp::class) {
            $names = \array_merge($names, [
                // Microsoft Office $1
                'Microsoft Office Access', 'Microsoft Office Excel', 'Microsoft Office OneDrive for Business',
                'Microsoft Office OneNote', 'Microsoft Office PowerPoint', 'Microsoft Office Project',
                'Microsoft Office Publisher', 'Microsoft Office Visio', 'Microsoft Office Word',
                // Podkicker$1
                'Podkicker', 'Podkicker Pro', 'Podkicker Classic',
                // radio.$1
                'radio.at', 'radio.de', 'radio.dk', 'radio.es', 'radio.fr',
                'radio.it',  'radio.pl', 'radio.pt', 'radio.se',  'radio.net',
            ]);
        }

        \natcasesort($names);

        return \array_unique($names);
    }   
```